packages/transformer/src/ECReferenceTypesCache.ts

```diff
@@ -221,6 +222,42 @@ export class ECReferenceTypesCache {
       ECReferenceTypesCache.bisRootClassToRefType[sourceRootBisClass.name];
     const targetType =
       ECReferenceTypesCache.bisRootClassToRefType[targetRootBisClass.name];
+
+    const sourceCAs = sourceClass.getCustomAttributesSync();
+    const targetCAs = targetClass.getCustomAttributesSync();
+    for (const [customAttributeName, _customAttribute] of sourceCAs) {
+      /*
+       * Since queryViews are typically associated with abstract classes and have no specific table in the database, we can ignore them.
+       */
+      if (
+        customAttributeName.toLowerCase() === "ecdbmap.queryview" &&
```

Comment:
Worried about potential performance implications, if I am checking the customattributes of every relationshipClass for the queryView attribute, will that be slow? Potential optimization is to only check if the sourceType or targetType are undefined? 

I imagine this is probably quite a small %age of any given transformer run, so maybe it doesn't matter?

Reply:
Started timing it: Info | core-backend.IModelTransformer | Checked 1444 classes for QueryView custom attribute. This took 4ms.

Its maybe overkill.